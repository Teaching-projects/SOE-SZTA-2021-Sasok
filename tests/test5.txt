357
15
0.5
951
95
4.5