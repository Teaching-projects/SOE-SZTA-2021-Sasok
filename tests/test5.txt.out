Első egység HP-ja: Első egység DMG-je: Első egység AS-je: Második egység HP-ja: Második egység DMG-je: Második egység AS-je: 0.5
Nem érvényes értéket adott meg, adjon meg egy számot! 
